README.md

```diff
@@ -54,6 +54,15 @@ To install the binary run `make install`.
 
 **Note**: Requires [Go 1.13+](https://golang.org/dl/)
 
+## Running in docker
```

Comment:
@alexanderbez what do you think about placing here example of .env file, i mean that it style slightly different than config.toml, values should not come as "string", but straight as is. Is it straight-forward enough?

Reply:
I think that is a good idea!

Reply:
@alexanderbez please, check [out](https://github.com/fissionlabsio/juno/pull/18/commits/424d10ca71b75a3f6a427c224747ae2b0219131f)